src/Utilities/InputOutput.f90

```diff
@@ -1972,6 +1974,28 @@ subroutine get_line(lun, line, iostat)
         exit
       end if
     end do
+    !
+    ! -- look for undetected end-of-record with isolated CR or LF
+    linesize = len(line)
+    crlfcheck: do i = 1, linesize
+      if (line(i:i) .eq. cr .or. line(i:i) .eq. lf) then
```

Comment:
I must be missing something fundamental here.  The outermost if statement looks for CR or LF in line(i:i).  Then, the inner if statements again check for CR or LF in line(i:i).  How does this check for an isolated CR or LF?  I'm sure you tested this so you know it works -- I just can't see why.

Reply:
The outer if statement identifies an isolated CR or LF. The inner if statement only prints the appropriate message for either CR or LF. Both CR and LF are not expected to be present in the middle of the line [line(i:i)]. If, for any reason, CR and LF appear in the middle of a line, that is still a bug.